### ISSUE TITLE
TensorRT 7 Resize produce inconsistent result with PyTorch Interpolate

### ISSUE THREAD

**Issue by xpngzhng** [onnx/onnx-tensorrt#457]
I have an HRNet semantic segmentation model trained with Pytorch, using interpolate bilinear align_corners=True
Then I convert it for C++ inference with align_corners=True and align_corners=False, I find that the output of the two mode are almost the same.

Using PyTorch1.5, onnx build from source (almost the latest version, showed 1.7.0), set interpolate with align_corners=False, export onnx model with opset version 10.

Then I use TensorRT 7.0.0.11 for inference in Windows. I find the output is quite different from those of PyTorch, the main problem is the result is almost a shifted and smaller version than the PyTorch version.

Is writing a plugin the only way to solve this problem?

Here are the result
torch align 
![torch_align_mask](https://user-images.githubusercontent.com/14209404/81463380-cf54ba80-91eb-11ea-9f9f-63e74ad0e006.png)
torch no align
![torch_mask](https://user-images.githubusercontent.com/14209404/81463387-e398b780-91eb-11ea-915b-666d82773c4d.png)
torch diff
![torch_diff](https://user-images.githubusercontent.com/14209404/81463412-0a56ee00-91ec-11ea-9eec-bae0c8cf39be.png)
trt no align
![mask](https://user-images.githubusercontent.com/14209404/81463396-f6ab8780-91eb-11ea-880c-bd2b3cae86ca.png)
trt torch diff
![torch_trt_no_align_diff](https://user-images.githubusercontent.com/14209404/81463408-04f9a380-91ec-11ea-9961-084e0070f146.png)

**Comment by shuo-zhao** [comment]
I met the same problem.
I am using trt7.1 and inference a model, the result is some pixels movement difference from onnx model inference. And the onnx inference is good to align with the tensorflow inference results.
Is there any settings to make upsample functions the same?

**Comment by sicccs** [comment]
Man, I have the same problem.
Have you solved it?

**Comment by sicccs** [comment]
I have the same problem.
Have you solved it?

**Comment by donrax** [comment]
The issue is connected to the interpolate (resize) layer in Pytorch. Also, take a look at https://github.com/onnx/onnx-tensorrt/issues/361#issuecomment-654278333

You can try and solve it by using the following:
`x = F.interpolate(x, size=(H, W), mode='bilinear', align_corners=True)`

Pytorch 1.5.1
Tensorrt 7.1.3 ([nvidia doc](https://docs.nvidia.com/deeplearning/tensorrt/release-notes/tensorrt-7.html))
Opset 11

**Comment by kevinch-nv** [comment]
@xpngzhng thanks for the report. Can you clarify a few things to help us narrow down the issue?

- Is this reproducible on the latest TRT version? We've made some updates on what resize modes of ONNX we actually support.
- Is this reproducible on ONNX opset >= 11? ONNX has increased their range of resize modes since opset 11, and that may the cause of the discrepancy seen between Pytorch and ONNX.

**Comment by kevinch-nv** [comment]
Closing this issue due to inactivity - please open a new issue if the problem persists.
